addons/account/wizard/account_move_send.py

```diff
@@ -676,6 +676,10 @@ def _process_send_and_print(self, moves, wizard=None, allow_fallback_pdf=False,
             if move.send_and_print_values:
                 move.send_and_print_values = False
 
+        # Update state so it shows 'sent' instead of 'posted'
+        for move in moves:
```

Comment:
I see that there is a loop just above the one you are adding that go through the moves so lets put the modification of the state there :smile: